packages/mobile/src/invite/saga.ts

```diff
@@ -182,6 +185,12 @@ export function* sendInvite(
       }
     )
 
+    if (features.ESCROW_WITHOUT_CODE && amount) {
```

Comment:
If `ESCROW_WITHOUT_CODE` is enabled, but `amount` is `0` or `undefined` we should set the user up to send the invite text, the exit the flow. If we continue, it will transfer 25 cents to a random address, which is less   than ideal.

Reply:
Good call